config/formats/javascript-export.ts

```diff
@@ -0,0 +1,24 @@
+
+import StyleDictionary from 'style-dictionary';
+import { format } from "prettier";
+import { jsonToNestedValue } from '../utilities/jsonToNestedValue';
+
+const { fileHeader } = StyleDictionary.formatHelpers;
+
+export const javascriptExport: StyleDictionary.Formatter = ({ dictionary, file, options, platform }) => {
```

Comment:
I can't tell the difference between this and the commonjs one 🙈. can we rename this to something more descriptive of what it is? Is this an ESM format?

Reply:
Yes, it just uses `export` instead of `module.export` so I guess `esm` would be fine. 👍